src/Traits/Rolable.php

```diff
@@ -18,7 +18,7 @@ public function getDataAttribute()
             'attributes'    => collect($this->attributes)->except($this->hidden),
             'relationships' => [
                 'role' => [
-                    'data' => $this->role ? collect($this->role->data)->except('relationships', 'attributes') : null,
```

Comment:
I do not want the Role attributes or relationships to be attached to the User. Just the ID and Type.